pkg/varlinkapi/system.go

```diff
@@ -76,7 +79,19 @@ func (i *LibpodAPI) GetInfo(call iopodman.VarlinkCall) error {
 		Graph_status:         graphStatus,
 	}
 
+	registriesInterface := info[2].Data["registries"]
+	insecureRegistriesInterface := info[3].Data["registries"]
```

Comment:
Not a change for this PR, but I'm assuming the array entries for info are used in more than one place.  It might be good to have variables defined for them in a common place.  So if things ever need to change, the numbering could be changed there.  Maybe something like:

INFO_REGISTRIES_INDEX := 2
INFO_INSECURE_INDEX := 3

or some such and then use the variables here in this code.  Just something to consider.